testing/testcafe/tests/dataGrid/columnReordering.ts

```diff
@@ -129,3 +129,46 @@ test('The separator should display correctly when dragging column', async (t) =>
   allowColumnReordering: true,
   allowColumnResizing: true,
 }));
+
+test('column separator should work properly with expand columns', async (t) => {
+  const dataGrid = new DataGrid('#container');
+  const columnSeparator = dataGrid.getColumnSeparator();
+
+  await t.drag(Selector('.dx-group-panel-item').nth(0), 0, 30);
```

Comment:
Let's not use selectors directly
Instead, we should add the getGroupPanel method to the DataGrid testing model